ando/tools/generator/AnDOGenerator.py

```diff
@@ -239,6 +239,46 @@ def generate_metadata_files(self):
                     create_file(mfile, parents[(3-level)] / mfile.name,
                                 mode='copy')
 
+    def create_metadata_file_participants(self) -> bool:
+        file = "participants"
+        exts = ['.csv', '.json']
+        for ext in exts:
+            filename = file + ext
+            try:
+                f = open(os.path.join(self.basedir, filename), 'w+')
```

Comment:
don't you overwrite the file anyway in 'w+' mode. Are you sure this raises a FileExistsError?

Reply:
Yes you right